examples/cluster/plot_adjusted_for_chance_measures.py

```diff
@@ -117,11 +117,10 @@ def fixed_classes_uniform_labelings_scores(
 plots = []
 names = []
 
-style.use("seaborn-colorblind")
+style.use("seaborn-v0_8-colorblind")
```

Comment:
This is failing with older matplotlib versions. We can adapt the code slightly

```python
import matplotlib as mpl
from sklearn.utils import parse_version

if parse_version(mpl.__version__) < parse_version("3.6"):
    colorblind_style = "seaborn-colorblind"
else:
    colorblind_style = "seaborn-v0_8-colorblind"

...

style.use(colorblind_style)
```

It should be better to add it on the top of the example

Reply:
A less "hacky" option is to set the color using seaborn diractly as follows:
```suggestion
sns.color_palette("colorblind")
```
Then you would have to change the line
```python
import matplotlib.style as style
```
to
```python
import seaborn as sns
```

Reply:
True, we are depending on seaborn in the example. It is a better fix. Thanks @ArturoAmorQ.

Reply:
Thanks both @glemaitre and @ArturoAmorQ!
I have changed this part to `import seaborn as sns` and `sns.color_palette("colorblind")`.